game/lib/audio.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:audioplayers/audio_cache.dart';
```

Comment:
Why don't you use `Flame.audio` here?

Reply:
Because it only works with audio cache for now

Reply:
Flame audio uses audio cache: https://github.com/flame-engine/flame/blob/master/lib/flame_audio.dart